packages/ui-phrases/README.md

```diff
@@ -0,0 +1,11 @@
+# `@logto/ui-phrases`
```

Comment:
Change here accordingly if you’d like to change the package name

Reply:
updated